test-support/page-object/properties/count.js

```diff
@@ -22,6 +22,7 @@ export function count(selector, options = {}) {
     isDescriptor: true,
 
     get() {
+      options.multiple = true;
       return findElement(this, selector, options).length;
```

Comment:
We can use `Ember.$.merge({}, options, { multiple: true })` here so we don't mutate client objects